src/shadowbox/integration_test/run_action.sh

```diff
@@ -26,7 +26,9 @@ declare -i result=0
 
 if ./test.sh > "${LOGFILE}" 2>&1 ; then
   echo "Test Passed!"
-  rm "${LOGFILE}"
+  # Removing the log file sometimes fails on Travis.  There's no point in us cleaning it up
+  # on a CI build anyways.
+  [[ -z TRAVIS_JOB_ID ]] || rm "${LOGFILE}"
```

Comment:
The integration test must not be aware of travis. It makes it harder to reason about it.
There must be some race condition?
In any case, change to `rm -f` and that will not fail if the file doesn't exist.

```suggestion
  rm -f "${LOGFILE}"
```